keras_cv/layers/attention_pooling.py

```diff
@@ -0,0 +1,33 @@
+from keras import layers
+
+
+class AttentionPooling(layers.Layer):
+    """Implements the Pooled Attention Layer used in "coca": Contrastive Captioners are Image-Text Foundation Models"
+    (https://arxiv.org/pdf/2205.01917.pdf), consisting of a Multiheaded Attention followed by Layer Normalization.
```

Comment:
You probably need to rename this pooling layer for COCO. There are many ways to implement attention pooling layer and so this could be confusing later. Or, you can define a separate layers dedicated for coco models.

layers/coco_layers.py

define AttentionPooling in the coco_layers.py